src/decode.rs

```diff
@@ -4619,7 +4618,10 @@ pub(crate) unsafe fn rav1d_decode_frame_init_cdf(
     let frame_hdr = &***f.frame_hdr.as_ref().unwrap();
 
     if frame_hdr.refresh_context != 0 {
-        rav1d_cdf_thread_copy(f.out_cdf.data.cdf, &f.in_cdf);
+        rav1d_cdf_thread_copy(
+            &mut f.out_cdf.cdf().unwrap().try_write().unwrap(),
```

Comment:
`cdf().unwrap()` and `refresh_context != 0` are checking the same thing, so this can be an `if let Some`.

Reply:
Yes, but I have changed `cdf()` to `cdf_write()` which tries to lock the field and requires an unwrap anyway.